cset-workflow/includes/metplus_grid_stat.cylc

```diff
@@ -0,0 +1,16 @@
+{% if METPLUS_GRID_STAT %}
+[scheduling]
+    [[graph]]
+    {{CSET_CYCLE_PERIOD}} = """
+    metplus_grid_stat => housekeeping_full
+    """
+
+[runtime]
+    [[metplus_grid_stat]]
+    # Runs METplus wrappers for point stat calculations.
+        inherit = METPLUS
+        [[[environment]]]
+        METPLUS_FCST_DIR = {{METPLUS_FCST_DIR}}
+        METPLUS_ANA_DIR = {{METPLUS_ANA_DIR}}
+        ROSE_APP_OPT_CONF_KEYS = {{METPLUS_OPT_CONFIG_KEYS}}
```

Comment:
Should these environment variables be moved into the `METPLUS` family task? It seem that most will use them.

Reply:
Moved to `METPLUS` family in `flow.cylc`. Note that the `METPLUS` family is also defined in the `site/{site}.cylc` file and holds the site-specific information to run METPlus (i.e. platform and directives).

Reply:
I think that should merge fine, and have both the variables from the flow.cylc, and the directives from the site config.